org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/XbaseCompiler.java

```diff
@@ -186,25 +187,35 @@ protected void _toJavaExpression(XListLiteral literal, ITreeAppendable b) {
 				if (literalType instanceof CompoundTypeReference) {
 					for (LightweightTypeReference c : literalType.getMultiTypeComponents()) {
 						if (c.isArray()) {
-							b.append("new ")
-							.append(c.getType()) // append raw type since we cannot create generic arrays
-							.append(" ");
+							b.append("new ").append(c.getType()) // append raw
+																	// type
```

Comment:
Format all the things? What did happen here?

Reply:
Damn. Eclipse auto-format fail.